src/main/java/com/hivemq/client/internal/mqtt/handler/publish/incoming/MqttIncomingPublishService.java

```diff
@@ -61,13 +60,12 @@
     }
 
     @CallByThread("Netty EventLoop")
-    void onPublishQos0(final @NotNull MqttStatefulPublish publish, final int receiveMaximum) {
-        if (qos0Queue.size() >= (2 * receiveMaximum)) { // TODO receiveMaximum
+    void onPublishQos0(final @NotNull MqttStatefulPublishWithFlows publishWithFlows, final int receiveMaximum) {
+        if (qos0Queue.size() >= receiveMaximum) { // TODO receiveMaximum
```

Comment:
isnt this TODO resolved?

Reply:
Actually not, there should be an additional configuration option for the QoS 0 queue limit. Currently the same as the receiveMaximum is used